docs/quickstart.rst

```diff
@@ -110,13 +112,16 @@ Using `HTTPie <http://httpie.org>`_, it is as easy as:
 
 ::
 
-    $ http -v http://localhost:8000/v0/ --auth user:pass
+    $ http -v http://localhost:6543/v0/ --auth user:pass
 
+.. note::
+
+    In the case of *Basic Auth*, there is no need of registering a user/password. Just pick any combination you want, and include them in each request.
```

Comment:
Two notes: we try to avoid the use of "you" in the documentation, and we try to wrap lines to 80 chars in the documentation as well.